frontend-v2/src/app/admin/page.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image";
+import { get } from "@/utils/request";
+import { Reviews, Reports } from "@/types/api";
+import AdminContent from "@/components/AdminContent/AdminContent";
+
+export default async function () {
```

Comment:
`const session = await getServerSession(authOptions);`

`if (!session) redirect them to home page;`

if session is there, use the GET /user/{zid} - if in that response if the person isn't an admin redirect them to home